lib/ClangImporter/ImportDecl.cpp

```diff
@@ -1774,46 +1805,70 @@ namespace {
 
       //
       // Create a computed value variable
-      auto computedVar = new (cxt) VarDecl(/*static*/ false,
-                                           /*IsLet*/ false,
-                                           SourceLoc(), computedVarName,
-                                           bridgedType, structDecl);
+      auto computedVar =
+          new (cxt) VarDecl(/*static*/ false,
+                            /*IsLet*/ false, SourceLoc(), computedVarName,
+                            bridgedType, structDecl);
```

Comment:
Please put the formatting back the way it was. I don't know what clang-format is trying to do, but it got it wrong.